compiler/cpp/test/lexer/LexerTest.cpp

```diff
@@ -0,0 +1,447 @@
+//
+// Created by Marc on 30.05.2021.
+//
+
+#include <vector>
+#include <lexer/Lexer.h>
+#include <gtest/gtest.h>
+#include "../util/LanguageDescriptor.h"
+#include "../util/FileReader.h"
+
+struct LexerParams {
+    const std::string fileName;
+    std::vector<TokenType> expectedTokensLineComment;
+    std::vector<TokenType> expectedTokensBlockComment;
+};
+
+const std::vector<LanguageDescriptor> LANGUAGES {
+    LanguageDescriptor("yml", "#", "", ""),
+    LanguageDescriptor("cpp", "//", "/*", "*/"),
+    LanguageDescriptor("html", "", "<!--", "-->"),
+    LanguageDescriptor("py", "#", R"(""")", R"(""")")
```

Comment:
you're right, the preferred identifier is `"""`, not `'''` , see [PEP 257](https://www.python.org/dev/peps/pep-0257/)

Reply:
Yes. I also wondered why Python tutorial pages often do not even mention that there are block comments supported. Some even say that block comments are not supported natively. Maybe they added support for those with a more recent version?!

Reply:
Thanks for pointing this out, I always thought that those were actual comment identifiers. But they are not!

> Since Python will ignore string literals that are not assigned to a variable, you can add a multiline string (triple quotes) in your code, and place your comment inside it:

[from W3Schools](https://www.w3schools.com/python/python_comments.asp)

I think one can still consider them to be multline comment identifiers ;)

Reply:
but it is possible to assign them to a variable, e.g. 

```python
a = """
    test
    """
```

Reply:
Oh, yes. I remember to have read that somewhere. This are only some string literals hanging around unused in the source code 😆